firefox-ios/Client/pings.yaml

```diff
@@ -91,3 +91,26 @@ usage-reporting:
     include_info_sections: false
     ping_schedule:
       - baseline
+
+onboarding-opt-out:
```

Comment:
hi @travis79 I was wondering if you think the structure of this ping is correct.